.github/PULL_REQUEST_TEMPLATE/pull_request_template_bug-fix.md

```diff
@@ -0,0 +1,48 @@
+<!-- Please, fill in the description as completely as possible.-->
+
+### Description
+
+<!-- Explain the issue with the bug (what part of the code, what are the side effects of the bug)
+       How did the bug was found, do you know what commit introduced the bug? -->
```

Comment:
The extra spaces are made on purpose since the markdown code on Github is not monospace and I wanted it flushed with the first line